CHARTER.md

```diff
@@ -0,0 +1,237 @@
+
+The TSC may, at its discretion, invite any number of non-voting observers to participate in the public portion of TSC discussions and meetings.
+
+A TSC member may be removed by vote from the TSC if, during a 3-month period, all of the following are true:
+
+- They do not participate in any TSC related discussions or votes
+- They do not provide any form of excuse or no excuse is known for their absence
+
+The initial TSC members are @relequestual, @awwright, @gregsdennis, @Julian, @jdesrosiers, @karenetheridge, with @relequestual being the initial chair.
+
+## Section 4: Roles & Responsibilities
+The JSON Schema project is jointly governed by a Technical Steering Committee (TSC) which is responsible for high-level guidance of the project.
+
+The TSC has final authority over this project including:
+
+- Technical direction
+- Project governance and process (including this policy)
+- Contribution policy
+- GitHub repository hosting and administration
+- Establishment of and delegation to working groups or teams
+- Mediating technical conflicts
+- Mediating non-technical conflicts (until a formal Code of Conduct Committee is established)
+
+In joining the TSC, members commit to communicate on a regular basis and respond to issues raised by the TSC in a timely manner. If they are no longer able or willing to make such a commitment, they should discuss this with the TSC or a TSC Chair.
+
+### Section 4.1 Project Operations & Management
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### Section 4.2: Decision-making and Voting
+The TSC follows a formal consensus seeking decision making model.
+In some situations, a vote may be preferable, however a vote will not be used to make the vast majority of decisions.
+In the unlikely case where it seems that consensus cannot be reached after multiple attempts, the decision process may be moved to resolve via a vote. This is not expected to happen, but defined as a backup.
+
+#### Decision-making via consensus
+
+When a TSC member looks for an issue to be discussed and a decision to be made, they must start by assessing if they feel it warrants potentially being a non-public discussion and decision. The TSC member must reach out to the TSC chairs to request a non-public discussion and decision. A discussion and decision will be non-public at the discretion of the TSC chairs. It is expected that decision discussions and voting will only be non-public in rare circumstances.
+
+Both the Quick and Standard consensus process require the creation of an Issue in the TSC GitHub repository.
+The Issue must:
+- Include brief introductory information about the decision that needs to be made
+- Be labelled with 'tsc-decision'
```

Comment:
```suggestion
- Be labeled with `tsc-decision`
```

"Labeled" is the US standard.  [Ref](https://www.scribbr.com/us-vs-uk/labelled-or-labeled/)

We should probably agree to use either a UK or US standard for official documents (this, the spec, the website, documentation, etc.).

Also, you've used backticks for code formatting later for the `tsc-vote` label (line 192).  Changing here for consistency.

Reply:
I've tried to use US english. I don't always get it right though. Thanks for picking this up.